### PR TITLE
Add HDD Testing in Production HOL and media images

### PR DESCRIPTION
**Pull request by sawpresto**
Add HDD Testing in Production hands-on lab.

### CODE REVIEW

**Comment by sawpresto** [comment]
I think that you should be able to see the commit that I just pushed up now, "Change deployment link from master to aspnet45?" Looks like it shows up right under the first commit. :) Let me know what you think.

**Comment by sawpresto** [comment]
Pushed another commit to blur out name and email.

File: docs/HOL_HDD_Testing_in_Production/HDD Testing in Production with Azure Websites HOL.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+![](<media/step23.png>)
+
+**Step 24.** After publication is complete, a browser window should open and navigate to the homepage. Verify that the test site waits for ten seconds before transitioning between images in the carousel.
+
+![](<media/step6.png>)
+
+**Step 25.** Close the browser window.
+
+
+###Task 2: Adding Route Rules to Direct Traffic to the Test Site with PowerShell
```
Comment (EMaher) [inline docs/HOL_HDD_Testing_in_Production/HDD Testing in Production with Azure Websites HOL.md]: You can also do this on the portal.  I believe its under 'All Settings' then 'Traffic Routing'

Comment (dtzar) [inline docs/HOL_HDD_Testing_in_Production/HDD Testing in Production with Azure Websites HOL.md]: Correct - this is good to show how you can do it in both places.  Later on in the HOL they show the equivalent in the portal. Please keep as-is.

Comment (EMaher) [inline docs/HOL_HDD_Testing_in_Production/HDD Testing in Production with Azure Websites HOL.md]: Ah.  I see that now.

**Pull request merged**